db/migrate/20210721064506_add_mentoring_stats_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddMentoringStatsToUser < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :num_solutions_mentored, :integer, limit: 3, null: false, default: 0
+    add_column :users, :satisfaction_rating, :integer, limit: 1, null: false, default: 100
```

Comment:
The satisfaction rating is a value between 0 and 100, so it is actually a percentage. I consided renaming it, but was unsure if `satisfaction_rating_percentage` would be better.

I also considered making this a nullable field, but I've opted for not doing that and defaulting to 100% satisfaction rating instead. Let me know what you think.

Reply:
I think naming it as `mentor_satisfaction_percentage` and having it as `nullable`. Sorry! 🙂 

Then in `app/helpers/view_components/mentor/header.rb` only showing it if it's not null.

Reply:
(I think if someone's not done any mentoring we want to be able to know that via this rating too, basically)